kube-scheduler-healthcheck

```diff
@@ -10,23 +12,36 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: ${TEST_POD_NAME}
+  labels:
```

Comment:
since we're labelling this, could we either label it with which pod owns this health check or else add an owner reference, too?

Reply:
I wanted to add that, but I'm not sure the best way to find your own pod name without passing in as env var.

It also seemed like a separate change.

Reply:
Ohhh. You know, we really should just add ownerReferences to fix this bug.

Then the standard kubernetes garbage collection will clean the pods up once the parent scheduler pod is deleted.